hipe4ml/plot_utils.py

```diff
@@ -617,6 +618,54 @@ def plot_feature_imp(df_in, y_truth, model, labels=None, n_sample=10000, approxi
     return res
 
 
+def plot_optuna_results(study, params=None, target_name='Objective value', outfile_names=None):
+    """
+    Visualize optuna optimization: history, hyperparameters importance,
+                                   parallel coordinates, hyperparameters contour plot
+
+    Parameters
+    -------------------------------------------
+    study: optuna.study.Study
+        Optuna study object
+
+    params: list
+        List of parameters to be plotted.
+        If None all the parameters are plotted.
+
+    target_name: string
+        Name to display on the axis label and the legend
+
+    outfile_names: list
+        List of strings containing the names of the output files.
+        If None, the files are named with default names.
+
+    Returns
+    -------------------------------------------
+    out: list of matplotlib.figure.Figure and titles
+    """
+    fig = []
+    if outfile_names is None:
+        outfile_names = ['Optimization_History', 'Optuna_Hyperpars_Importances',
+                         'Optuna_Parallel_Coordinates', 'Optuna_Contour']
+    else:
+        outfile_names = [outfile_names[0], outfile_names[1], outfile_names[2], outfile_names[3]]
+
+    fig.append(optuna.visualization.plot_optimization_history(study, target_name=target_name))
+    fig.append(optuna.visualization.plot_param_importances(study, params=params, target_name=target_name))
```

Comment:
Here there is an additional parameter `evaluator` that I think should be nice to be able to change
https://optuna.readthedocs.io/en/stable/reference/visualization/generated/optuna.visualization.plot_param_importances.html#optuna.visualization.plot_param_importances

Reply:
Yes, it could be nice to look also into this. However, I chose to avoid this parameter since I don't know how many people will really use it in the end. Anyhow,  if you think it is worth having it I will include it.

Reply:
I would include it. Through the package, we try to give full access to the option of external functions we use. In case, you could add this as a `kwargs` if you think is better to not add it explicitly.